configure.ac

```diff
@@ -64,13 +64,22 @@ AC_ARG_ENABLE([esapi],
             [enable_esapi=$enableval],
             [enable_esapi=yes])
 
-
 AC_ARG_ENABLE([esapi], 
     AS_HELP_STRING([--disable-esapi], [Enable TSS ESYS API library @<:@default=yes@:>@]),
     [],
     [enable_esapi="yes"])
 AM_CONDITIONAL(ESAPI, test "x$enable_esapi" = "xyes")
 
+AC_ARG_ENABLE([tcti-device-async],
+    AS_HELP_STRING([--enable-tcti-device-async],
+	           [Enable asynchronus operation on TCTI device
+		    (note: This needs to be supported by the kernel driver). default is no]),
+    [enable_tcti_device_async=$enableval],
+    [enable_tcti_device_async=no])
+AM_CONDITIONAL(TCTI_ASYNC, test "x$enable_tcti_device_async" = "xyes")
```

Comment:
I don't see a place in the makefile where you use this variable. Unless I missed it, this isn't necessary.

Reply:
Removed